doc/status-code/api/status-codes.rst

```diff
@@ -205,7 +205,9 @@ Error codes
 
    .. summary:: A status code that indicates that a handle parameter is not valid.
 
-   A function can return this error any time a handle parameter is invalid.
+   A function can return this error any time it expects a parameter to be a handle to an existing object, but the handle is invalid.
```

Comment:
Prefer 'resource' to 'object' - consistent with the other error code descriptions.